warn/scrapers/mi.py

```diff
@@ -80,6 +79,23 @@ def scrape(
             pdf_file = cache.download(cache_key, pdf_url)
         pdf_list.append(pdf_file)
 
+    # parse pdf data
+    pdf_data = []
+    print(pdf_list)
+    # Parse the pdfs from 2007-2015
+    for file in range(len(pdf_list) - 9, len(pdf_list))[::-1]:
+        with pdfplumber.open(pdf_list[file]) as pdf:
+            for i in pdf.pages:
+                pdf_data += process_pdf_2(i.extract_text())
```

Comment:
Can we give these function more literate names than `_1` and `_2`?

Reply:
They are for the two types of processing we have to do for 2000-2006 pdfs and 2007-2015 pdfs. Renamed accordingly.

Reply:
Michigan is being removed again. Can we take out any edits to it entirely?